app/services/core_data_connector/import/taxonomies.rb

```diff
@@ -0,0 +1,94 @@
+module CoreDataConnector
+  module Import
+    class Taxonomies < Base
+      def cleanup
+        super
+
+        execute <<-SQL.squish
+          UPDATE core_data_connector_taxonomies
+             SET z_taxonomy_id = NULL
+        SQL
+      end
+
+      def load
+        super
+
+        execute <<-SQL.squish
+          UPDATE core_data_connector_taxonomies taxonomies
```

Comment:
Could we double check the formatting on these SQL queries to make sure the alignment is consistent with the other models?

Reply:
I've done a side-by-side comparison but it wasn't that many changes. Does RubyMine make it easier to do this type of formatting comparison? VSCode doesn't check formatting in `SQL.squish` blocks at all so the differences are pretty subtle to me.

Reply:
RubyMine doesn't make it any easier, I typically just format them manually. And looking back at some of the other importers, there are some inconsistencies in my SQL statements as well. This looks good though. I'll put together a high-level "How we should format SQL" and include it in the README so we can have consistency going forward. I'm not sure there's a "right" way to do it, as long as it's readable and consistent.